### PR TITLE
improve testing around amazon linux

### PR DESCRIPTION
**Pull request by chris-rock**
This builds on top of #194 and adds testing for Amazon Linux 1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,17 @@ It will not:
 * Cookbooks:
   * Sander van Zoest sysctl `https://github.com/svanzoest-cookbooks/sysctl`
 
+### Platform
+
+- Debian 6, 7, 8
+- Ubuntu 12.04, 14.04, 16.04
```
Comment (artem-sidorenko) [inline README.md]: Can we maybe remove the ubuntu 12.04 and centos 5? They are EOL

Comment (artem-sidorenko) [reply README.md]: Fedora 24 and debian 6 are EOL too

Comment (chris-rock) [reply README.md]: Sure, I update and remove them from metadata, too

Comment (chris-rock) [reply README.md]: We can update the metadata in the next major version, but I am not mentioning the old system in the readme.

Comment (artem-sidorenko) [reply README.md]: I would suggest to remove them from metadata too: I’m not sure if cookbook will work properly on this old distros at all. Its in my eyes only a leftover, which I missed to remove in the PRs.

Comment (chris-rock) [reply README.md]: I am going to do that in another PR if you're okay with that.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,17 @@ It will not:
...
   * Sander van Zoest sysctl `https://github.com/svanzoest-cookbooks/sysctl`
 
+### Platform
+
+- Debian 7, 8
+- Ubuntu 14.04, 16.04
+- RHEL 6, 7
+- CentOS 6, 7
+- Oracle Linux 6, 7
+- Fedora 24, 25
```
Comment (artem-sidorenko) [inline README.md]: It should be 26 and 27

Comment (chris-rock) [reply README.md]: Will fix that.

Comment (chris-rock) [reply README.md]: I move this to the other PR to, so that this PR focus on amazon linux

Comment (artem-sidorenko) [reply README.md]: Great!

**Comment by chris-rock** [comment]
All the metadata/platform updates moved to #204

**Pull request merged**